keras_cv/layers/preprocessing/random_rotation.py

```diff
@@ -78,6 +78,8 @@ class RandomRotation(BaseImageAugmentationLayer):
       bounding_box_format: The format of bounding boxes of input dataset. Refer
         https://github.com/keras-team/keras-cv/blob/master/keras_cv/bounding_box/converters.py
         for more details on supported bounding box formats.
+      segmentation_classes: The number of classes in the input segmentation mask.
+        Required iff augmenting data with sparse (non one-hot) segmentation masks.
```

Comment:
We should also make it clear whether the number of classes contain the 0 (background).